File: langs/freepascal/problems/load_balancer/stub.pas
{ Simulation stub. Fill methods from the problem spec.
  add_backend(backend_id)
  route()
  set_health(backend_id, flag)
  set_weight(backend_id, weight)
  sticky(client_id)
  done(backend_id) }

unit solution;

{$mode objfpc}{$H+}
{$M+}

interface

uses
  SysUtils;

type
  Simulation = class
  end;

function NewTarget: TObject;

implementation

function NewTarget: TObject;
begin
  Result := Simulation.Create;
end;

end.

File: langs/haskell/problems/load_balancer/stub.hs
-- Simulation stub. Fill methods from the problem spec.
-- add_backend(backend_id)
-- route()
-- set_health(backend_id, flag)
-- set_weight(backend_id, weight)
-- sticky(client_id)
-- done(backend_id)
module Solution (Simulation, newTarget) where

import Harness

data Simulation = Simulation

newTarget :: Simulation
newTarget = Simulation

instance Target Simulation
